_posts/blog/2018-10-18-review-living-archive.md

```diff
@@ -0,0 +1,20 @@
+
+There is evidence of an accountability and ethical responsibility to the indigenous peoples of the Northern Territory from the first time one enters the website. A notice that reads “Respecting ownership” is at its gate. It indicates that the contents belong to the “Aboriginal language owners, creators of the materials and their descendants.” The notice also serves as a warning to the Aboriginal and Torres Strait Islander users about the inclusion of “names, images, and voices of people who have passed away”. Once entered the website, browsing is very easy. Apart from the mentioned interactive map organized by places and languages, there is a menu bar with different categories for searching purposes: “Place”, “Language”, “People” (authors and illustrators), and “Books”. After a selection has been made, the new page shows the covers of the books. Some of them are very colorful, and most of them have a naïve esthetic meant for attracting the attention of children. The metadata is very thorough –title, author, illustrator, place, year of publication, abstract, description/genre, and others–, and it is followed by a “Right Statement”. This announcement entails a decolonial practice, that of contacting the creators for permission to publish, but also a call for users to reach out if they feel the book should not be shown. All the documents are available for exploring in PDF format, and for downloads. The website has a Creative Common License that privileges the sharing of the material.
+
+To sum up, or more accurately to start the conversation some conclusions are in hand. It is important to acknowledge that the Living Archive is not community-based (following Michelle Caswell’s reflections), which would be a step forward in the decolonization of indigenous knowledges. It is not the case because the archive was designed, and today is managed mostly by powerful knowledge-making universities external to the communities. The website is hosted in the Charles Darwin University Library’s open source Fez and Fedora based research repository. That being said, the Living Archive is a good community-centered model for educational institutions from all over the world that can gather the resources, have the infrastructure and hire the staff to make a project like this possible. Finding the books, searching for the creators, scannig the materials takes time, effort, and a special attitud towards the sensible significance they hold. Then, what is at stake in the making of this digital collection? At stake is the postcustodial turn of archival work, that in the context of indigenous materials means giving back. This collection shows a dedicated, thorough, and ethically responsible work looking after indigenous knowledges that puts the creators in the center by recognizing their ownership and negotiating control. The Living Archive opens the doors to thinking of postcustodial practices for indigenous collections in diverse, even radical, decolonial ways. 
```

Comment:
Again, you might break this into two paragraphs, starting with "what is at stake..."